testing/helpers/scheduler/helpers.js

```diff
@@ -20,6 +20,8 @@ export const CLASSES = {
     scrollableAppointmentsContainer: '.dx-scheduler-scrollable-appointments',
     schedulerSmall: '.dx-scheduler-small',
 
+    dataTableCell: '.dx-scheduler-date-table-cell',
```

Comment:
I suggest to call it `dateTableCell` - for consistency reasons

Reply:
Done